spec/dummy/app/controllers/inertia_render_test_controller.rb

```diff
@@ -17,4 +17,15 @@ def view_data
   def component
     render inertia: 'TestComponent'
   end
+
+  def lazy_props
+    render inertia: 'TestComponent', props: {
+      name: 'Brian',
+      sport: InertiaRails.lazy('basketball'),
+      level: InertiaRails.lazy do
+        'worse than he believes'
+      end,
+      grit: InertiaRails.lazy(->{ 'intense' })
+    }
+  end
```

Comment:
This is the API for using lazy props... it works with a plain value, a block, or a block/lambda passed as a value

Reply:
@reinink @bknoles please add this to the docs, had to dig a bit to find the api after seeing it in the changelog https://inertiajs.com/partial-reloads

Reply:
Oops, just saw the PR to update the Inertia site, nevermind me ☺️

Reply:
Yep! Awaiting merge right here: https://github.com/inertiajs/inertiajs.com/pull/143

😄